source/includes/_prices.md

```diff
@@ -1,13 +1,15 @@
 # Prices
 
-_Update 15 Jan 2019: implemented but not saved to a database_
-
-Price data of a set of assets / symbols (from_symbols) in a chose currency / symbol (to_symbol), gathered through our price aggregation.
+_Update 16 Jan 2019: implemented but not saved to a database_
 
 Currently it calls the source(s) in real-time, but soon we will start caching the answers and do scheduled refreshes
 
+## Cryptocurrencies
+
+Price data of a set of cryptocurrency symbols (symbols) in a chose currency / symbol (toSymbol), gathered through our price aggregation.
+
 ```shell
-curl "http://api.lettuce.money/prices?to_symbol=ZAR&from_symbols=BTC,USD,MSFT"
+curl "http://api.lettuce.money/pricing/cryptos?toSymbol=ZAR&symbols=BTC,ETH,XMR"
   -H "Authorization: mypersonalapikey"
 ```
```

Comment:
I highly recommend that we change this so that it's "toSymbol" and "fromSymbols" for consistency.
Also, I suggest that we change "cryptos" to "cryptocurrencies". But, over the long term, shouldn't it just be "/pricing/toSymbol..."? Why would you specify the asset type, when over the long term, that would be taken care of automatically?

Reply:
Changed, thanks.